app/models/beta/search/direct_hit.rb

```diff
@@ -0,0 +1,56 @@
+module Beta
+  module Search
+    class DirectHit
+      attr_accessor :goods_nomenclature_class,
+                    :goods_nomenclature_item_id,
+                    :producline_suffix
+
+      def id
+        "#{goods_nomenclature_item_id}-#{producline_suffix}"
+      end
+
+      def method_missing(method_name, *_arguments)
```

Comment:
There's a bunch of uninteresting (for redirecting) goods nomenclature stuff that I'm skipping over adding specific readers/default values for by using this callback